codec/unknownproto/doc.go

```diff
@@ -1,24 +1,22 @@
-/*
```

Comment:
why not leave this as /*?

Reply:
when I was reading the code, I found that there are two styles of multi-line comments in file descriptions, namely '/**/' and '//'. I chose the ‘//’ comment when adjusting, because I found that most of the multi-line comments using file descriptions in the project were ‘//’